### ISSUE TITLE
Mod not working for me.

### ISSUE THREAD

**Issue by Pudgemountain** [jetkai/Payday-2-Scarface-DLC#1]
I had downloaded and installed it but I don't get Scarface or the weapons. I already have the Mission DLC so I don't need that. I installed it correctly but Tony is not available and is not in the safehouse however it shows I have it installed.
![20230928212131_1](https://github.com/jetkai/Payday-2-Scarface-DLC/assets/146408783/fafeafc7-6dd3-4764-9fe9-1392d1936466)
![20230928212134_1](https://github.com/jetkai/Payday-2-Scarface-DLC/assets/146408783/71ff118d-aa66-402b-ab40-a321a8fedd1b)
![20230928212044_1](https://github.com/jetkai/Payday-2-Scarface
![20230928212720_1](https://github.com/jetkai/Payday-2-Scarface-DLC/assets/146408783/c818312f-31f5-4791-987a-64d24a4564a6)
-DLC/assets/146408783/b0568d94-9249-45e2-80cb-67f917262e76)

Sorry for the picture spam.

**Comment by Insertname855** [comment]
Im having the same issue here. Ive tried tinkering with it with no results. I have the icon in the mod page showing that it **is** installed. Hoping there's a fix or something I can do to get it to work. I don't wanna miss out on this character pack.

**Comment by ace-in-the-hole** [comment]
hi had the same issue is you add this mod into the mod floder (dont know if you need to take or keep this one (i keept it)) it should fix it and youll have tony and his wepones and perk deck                                                                                                                  the mod: https://github.com/pd2-stuff/scarface            hope this helped

**Comment by JoshDarkSoul** [comment]
Yeah I'm pretty sure that this mod doesn't work at all in 2026, besides unlocking Tony Montana in the safe house (can't even upgrade his room with Continental Coins so that it looks better btw) it doesn't unlock anything, and I know the mod is installed properly since it shows in the BLT screen in main menu... don't bother downloading it until it gets fixed, and I hope it does
